packages/shared/src/hooks/referral/useReferralCampaign.ts

```diff
@@ -41,7 +42,7 @@ const useReferralCampaign = ({
   campaignKey,
 }: UseReferralCampaignProps): UseReferralCampaign => {
   const featureFlag = campaignFeatureFlagMap[campaignKey];
-  const isCampaignEnabled = useFeatureIsOn(featureFlag);
+  const isCampaignEnabled = useFeatureIsOn(featureFlag) || !featureFlag;
```

Comment:
I think the condition should be specific whether it is the generic referral or a group of valid campaign keys. Reason being, the condition would now be true on any case.

Reply:
We can make it based on campaignkey valid? Would that work?

Reply:
Yes, I think that should work 🚀

Reply:
@sshanzel Changed the evaluation, let me know what you think?

Reply:
Looks amazing! 🤩